app/assets/src/styles/views/phylo_tree/phylo_tree_vis.scss

```diff
@@ -12,14 +12,9 @@
   }
 
   .node circle {
-    fill: $light-grey;
     opacity: 0.8;
```

Comment:
Can you keep it grey here? This is the default color.
Currently, we do not have a color if we disable the highlight per attribute.

Also, do you know which takes precedence fill in JS or in CSS?